### DISCUSSION TITLE
How many types can we open UiExplorer

### DISCUSSION THREAD

**Discussion by vajrang-b** [vajrang-b/RPA-Developer-in-30-Days#13] [Q&A] [answered]
list out all places where you can open uiexplorer

**Comment by usurabhi** [comment]
You can open the UI Explorer window, by 
1. clicking the UI Exloper button in the Selectors section, in Design tab
2. we can also launched it from the Tools page in the Studio backstage view  (It is available only if the UiPath.UIAutomation.Activities package is installed as a dependency to the project)

**Comment by Manjushree15** [comment, marked as answer]
1. It can be accessed by clicking on Ui Explorer button in the ribbon(Design Tab).
2. We can open it by clicking on Open in Ui Explorer option in the selector editor window.
3. We can also access it from Tools section, Available in the UiPath home page

**Comment by TiwariRakesh** [comment]
3 ways, 
1- choose from ribbon 
2- by selecting selector option under the target in properties panel
3 - can open from tools option in studio

**Comment by Ulhasguttikar** [comment]
1) Ui Explorer option from ribbon
2) using selector option under target property.
3) from studio backstage view
